lib/runner/test-runners/default.js

```diff
@@ -45,10 +45,8 @@ class DefaultRunner {
    * @return {Promise}
    */
   closeOpenSessions(err) {
```

Comment:
I believe that by refactoring in this way, it will be much cleaner.
```javascript
closeOpenSessions(err) {
    const inspectorServer = this.addtOpts?.globalsInstance?.inspectorServer;
    if (inspectorServer) {
        inspectorServer.closeSocket();
    }
}
```

Reply:
I guess, we can do this now since we are not supporting node 12 anymore

Reply:
It's supported from node 14 onwards

Reply:
yeah, looks good.

Reply:
updated